ultravox/data/dataset_config.py

```diff
@@ -11,7 +11,7 @@ class DataDictConfig(BaseModel):
     name: Optional[str] = None
     splits: List[str] = dataclasses.field(default_factory=list)
     num_samples: Optional[int] = None
-    total_samples: int
+    total_samples: int = 1
```

Comment:
@liPatrick for now I just made this change to allow the default config to run since all training was broken before AFAIK. Feel free to suggest a different method or modify in follow up PRs.

Reply:
I put up a PR to address this here: https://github.com/fixie-ai/ultravox/pull/104

Reply:
Perfect, thanks. I'll wait for that to merge then.